Tests/SentryTests/SentryCrash/SentryBinaryImageCacheTests.m

```diff
@@ -253,6 +255,31 @@ - (void)testCloseCacheWhileAdding
     XCTAssertEqual(result, 0);
 }
 
+// Adding a SentryBinaryImageCache test inside
+// SentryCrashBinaryImageCache to test integration between both
+// because is easier to control SentryCrashBinaryImageCache in an objc test
+- (void)testSentryBinaryImageCacheIntegration
+{
+    sentrycrashbic_startCache();
+
+    SentryBinaryImageCache *imageCache = SentryBinaryImageCache.shared;
+    [imageCache start];
+    // by calling start, SentryBinaryImageCache will register a callback with
+    // `SentryCrashBinaryImageCache` that should be called for every image already cached.
+    XCTAssertEqual(5, imageCache.cache.count);
```

Comment:
`m`:  Why is the image cache count precisely 5? Could that change in the future depending on the test setup? If yes, this test could easily break in the future.

Reply:
This is a magic number for tests. Im using 5 binary images for deterministic reason.

Reply:
I start a mock collection (lines 93-94) with 5 items that you be used to test `SentryBinaryImageCache`.
When I start `SentryBinaryImageCache` it should have a copy of those 5 initials items.

This 5 items represent 5 binary images loaded in the app.